### PR TITLE
Add CsTeamsCustomBannerText 

### PR DESCRIPTION
**Pull request by saleens7**

### CODE REVIEW

File: teams/teams-ps/teams/Get-CsTeamsCustomBannerText [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+## SYNTAX
+
+### Identity (Default)
+```
+Get-CsTeamsCustomBannerText [[-Identity] <String>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Returns all or a single instance of custom banner text(s).
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsCustomBannerText:26]:
```suggestion
Returns all or a single instance of custom banner texts.
```

File: teams/teams-ps/teams/Get-CsTeamsCustomBannerText [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+```powershell
+PS C:\> Get-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Gets the properties of the CustomText instance of TeamsCustomBannerText.
+
+## PARAMETERS
+
+### -Identity
+Policy instance name. Optional.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Get-CsTeamsCustomBannerText:48]:
```suggestion
Policy instance name (optional).
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+```powershell
+PS C:\> New-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Creates an instance of TeamsCustomBannerText with the name CustomText.
+
+## PARAMETERS
+
+### -Id
+The Identity of the CustomBannerText. You do not need to provide an Id as the backend will generate it for you. However if you wish to provide your own ID, you can provide your own GUID. Please note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText:40]:
```suggestion
The Identity of the CustomBannerText. You do not need to provide an ID as the backend will generate it for you. However, if you wish to provide your own ID, you can provide your own GUID. Note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```

File: teams/teams-ps/teams/New-CsTeamsCustomBannerText [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+Applicable: Microsoft Teams
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Text
+The text that the tenant admin would like to set in policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/New-CsTeamsCustomBannerText:55]:
```suggestion
The text that the tenant admin would like to set in the policy.
```

File: teams/teams-ps/teams/Remove-CsTeamsCustomBannerText [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+### Example 1
+PS C:\> Remove-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Removes a TeamsCustomBannerText instance with the name "CustomText".
+
+## PARAMETERS
+
+### -Identity
+Policy instance name. Optional.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Remove-CsTeamsCustomBannerText:38]:
```suggestion
Policy instance name (optional).
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+## SYNTAX
+
+### Identity (Default)
+```
+Set-CsTeamsCustomBannerText [[-Id] <Guid>] [<CommonParameters>]
+```
+
+## DESCRIPTION
+
+Updates a single instance of custom banner text.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText:25]:
```suggestion
Updates a single instance of a custom banner text.
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+### Example 1
+PS C:\> Set-CsTeamsCustomBannerText -Identity CustomText
+```
+
+Sets the properties of the CustomText instance of TeamsCustomBannerText.
+
+## PARAMETERS
+
+### -Id
+The Identity of the CustomBannerText. You do not need to provide an Id as the backend will generate it for you. However if you wish to provide your own ID, you can provide your own GUID. Please note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText:38]:
```suggestion
The Identity of the CustomBannerText. You do not need to provide an ID as the backend will generate it for you. However, if you wish to provide your own ID, you can provide your own GUID. Note that you have to provide a unique ID for every CsTeamsCustomBannerText you create.
```

File: teams/teams-ps/teams/Set-CsTeamsCustomBannerText [hunk 8]
```diff
@@ -0,0 +1,93 @@
...
+Applicable: Microsoft Teams
+Required: False
+Position: 1
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -Text
+The text that the tenant admin would like to set in policy.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Set-CsTeamsCustomBannerText:53]:
```suggestion
The text that the tenant admin would like to set in the policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@saleens7

**Comment by yogkumgit** [comment]
@tiburd As reviewers have already approved it, please help merging it after copy comments. Thanks

**Pull request merged**